tokens/transformTokens.js

```diff
@@ -4,6 +4,7 @@ const baseConfig = require('./config')
 // Register Web transforms
 require('./transformation/web')
 require('./transformation/tailwind')
+require('./transformation/skia')
```

Comment:
Nit: Since this isn't a web transform, maybe we should rename the comment above to "Register transforms", or something.